cpg_workflows/stages/realign_genotype_with_dragen.py

```diff
@@ -142,90 +140,137 @@ def queue_jobs(self, sequencing_group: SequencingGroup, inputs: StageInput) -> S
 
 
 @stage(required_stages=[PrepareIcaForDragenAnalysis, UploadDataToIca])
-class AlignGenotypeWithDragen(SequencingGroupStage):
+class ManageDragenPipeline(SequencingGroupStage):
     # Output object with pipeline ID to GCP
     def expected_outputs(
         self,
         sequencing_group: SequencingGroup,
-    ) -> cpg_utils.Path:
+    ) -> dict[str, cpg_utils.Path | str]:
```

Comment:
```suggestion
    ) -> dict[str, cpg_utils.Path]:
```

both these outputs are Paths